### PR TITLE
feat: config witnesses by file and dir args

### PR DESCRIPTION
**Pull request by kentbull**
This aligns `kli witness start` with `kli agent start` for the `--config-dir` and `--config-file` args.

While this is functional there is one problem. There are empty config files written to `$CONFIG_DIR/keri/cf` while the files in `$CONFIG_DIR/keri/cf/main` are the ones that must be populated. This is annoying and I would rather eliminate this behavior before this is merged.

### CODE REVIEW

**Comment by kentbull** [comment]
This resolved #456 for me.

**Comment by kentbull** [comment]
Looks like I was wrong. It was the arguments I was passing in to the kli init command that were creating the file in ${CONFIG_DIR} :
```
kli init --name wan --salt 0AB3YW5uLXRoZS13aXRuZXNz --nopasscode \
  --config-dir "${KERI_SCRIPT_DIR}" --config-file wan-witness
```
I was missing a main/ path segment on the wan-witness file I was using.
Due to the default arguments for the Configer class it uses the path segment "main" when looking for configuration files.

```
class Configer(filing.Filer):
  ...
  TailDirPath = "keri/cf"
  ...
   def __init__(self, name="conf", base="main", filed=True, mode="r+b",
                 fext="json", human=True, **kwa):
    ...
```

The problem can be worked around (solved?) by just putting a prefix on the path given to the --config-file argument as so:

```
--config-file main/wan-witness
```

There are two potential --base arguments though only one exists in the CLI interface for kli witness start right now.
1. The base directory for the keystore. (in the .keri or /usr/local dir)
2. The base directory for the configuration files. (set to $KERI_SCRIPTS_DIR in the sample scripts)

In the future we could separate them out with another change to the `kli witness start` command by allowing `--base-keystore` and `--base-config` options.

File: src/keri/app/cli/commands/witness/start.py [hunk 1]
```diff
@@ -54,13 +60,16 @@ def launch(args):
...
+               config_dir=args.config_dir,
+               config_file=args.config_file)
 
     logger.info("\n******* Ended Witness for %s listening: http/%s, tcp/%s"
                 ".******\n\n", args.name, args.http, args.tcp)
 
 
-def runWitness(name="witness", base="", alias="witness", bran="", tcp=5631, http=5632, expire=0.0):
+def runWitness(name="witness", base="", alias="witness", bran="", tcp=5631, http=5632, expire=0.0,
+               config_dir="", config_file=""):
```
Comment (pfeairheller) [inline src/keri/app/cli/commands/witness/start.py:72]: In KERIpy, camelcase is standard for parameters.  Can you update these two?

**Review by pfeairheller — CHANGES_REQUESTED** [review]

File: src/keri/app/cli/commands/witness/start.py (hunk 1 above)
Comment (kentbull) [reply src/keri/app/cli/commands/witness/start.py]: Yes, just pushed the update. I should get a pre-commit hook set up with a code style check.

**Review by pfeairheller — APPROVED** [review]

**Pull request merged**